packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -242,9 +242,45 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     );
   }
 
-  // Render Items
+  // Render items
   @autobind
   private _onRenderItem(item: IDropdownOption): JSX.Element {
+    if (item.text === '-') {
```

Comment:
I agree 👍 I thought that logic was a bit weird in ContextualMenu.  I'll remove this check.